source/Preferences.cpp

```diff
@@ -172,6 +172,9 @@ void Preferences::Load()
 	settings["Ship outlines in HUD"] = true;
 	settings["Extra fleet status messages"] = true;
 	settings["Target asteroid based on"] = true;
+	settings["Confirm 'Sell Outfits' button"] = true;
+	settings["Confirm 'Sell Specials' button"] = true;
+	settings["'Sell Outfits' without outfitter"] = true;
```

Comment:
I was going to say "I think the default for the "'Sell Outfits' without outfitter" option should be false", but then I realized you can pick up random crap floating through space and have no way to get rid of it...

I really like the idea of a dialog to confirm outfit sales.